samples/config/prod/s-cluster/userstore.yaml

```diff
@@ -1,31 +1,44 @@
 djInstance: userstore
 
-numberSampleUsers: "1000"
-
 resources:
   limits:
     memory: 6Gi
   requests:
     memory: 4Gi
 
-storageSize: "100Gi"
-storageClass: "fast"
+
 
 # Uncomment the next 3 line if you don't want to use cgroups which is the default JVM setting
 #opendjJavaArgs: >
 #  -server -Xms4g -Xmx4g -XX:+UseCompressedOops -XX:+UseG1GC -XX:+UseNUMA  
 #  -XX:MaxGCPauseMillis=100 -verbose:gc -Xlog:gc:/tmp/gc.log
 
+# For JDK 8
+opendjJavaArgs: >
+  -server -Xms4g -Xmx4g -XX:+UseCompressedOops -XX:+UseG1GC -XX:+UseNUMA 
+  -XX:MaxGCPauseMillis=100 -verbose:gc -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps 
+  -XX:+PrintGCDetails -XX:+PrintPromotionFailure -XX:+PrintAdaptiveSizePolicy -Xloggc:/tmp/gc.log
+  
+# For JDK 11
```

Comment:
Has the ds chart moved back to JDK 8?

Reply:
Hmm...good point..need to check with Dr. Strange.  If yes then i will fix in next PR.